.changelog

```diff
@@ -4,6 +4,7 @@
 # - Fixed error in Bat section of Upgrade Guide (bsc#1234567)
 # For guidelines: https://en.opensuse.org/openSUSE:Creating_a_changes_file_(RPM)
 
+- Added detailed information about invalidated credentials after PAYG instance get shutdown
```

Comment:
```suggestion
- Added warning about channel synchronization failure because of invalidated
  credentials in Connect PAYG instance section of Installation and Upgrade
  Guide
```